src/Form/CitizenProjectCommandType.php

```diff
@@ -117,30 +133,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             throw new InvalidConfigurationException('A pre set data is required in '.__CLASS__);
         }
 
-        $citizenProject = $command->getCitizenProject();
-
-        if (!$citizenProject instanceof CitizenProject || !$citizenProject->isApproved()) {
-            $builder
-                ->add('committees', CollectionType::class, [
-                    'required' => false,
-                    'entry_type' => TextType::class,
-                    'entry_options' => ['label' => false],
-                    'allow_add' => true,
-                    'allow_delete' => true,
-                    'by_reference' => false,
-                ])
-                ->add('committees_search', TextType::class, [
-                    'mapped' => false,
-                    'required' => false,
-                    'filter_emojis' => true,
-                    'attr' => [
-                        'placeholder' => 'Ajouter le comité local',
-                    ],
-                ]);
-
-            $builder->get('committees')->addModelTransformer($this->committeeTransformer);
-        }
-
         if ($command instanceof CitizenProjectCommand && $command->isCitizenProjectApproved()) {
```

Comment:
Not related but this if can be simplified to `if ($command->isCitizenProjectApproved()) {` due to the condition above.

Reply:
Thanks you @HeahDude, good catch !